core/domain/state_domain.py

```diff
@@ -1495,6 +1541,23 @@ def get_content_html(self, content_id):
 
         return content_id_to_html[content_id]
 
+    def is_rte_content_supported_on_android(self):
+        """Checks whether the RTE components used in the state is supported by
+        Android.
+
+        Returns:
+            bool. Whether the RTE components in the state is valid.
+        """
+        regex_string = r'oppia-noninteractive-'
+        for rte_id in android_validation_domain.StoryConstants.INVALID_RTE_COMPONENTS: # pylint: disable=line-too-long
```

Comment:
Avoid pylint-disable. Create a separate internal variable to alias the long constant name.

Reply:
Done